src/jit/compiler.h

```diff
@@ -1771,7 +1773,11 @@ class Compiler
     // a PSPSym for functions with any EH.
     bool ehNeedsPSPSym() const
     {
+#ifdef _TARGET_X86_
```

Comment:
The problem is that the CLR ABI doesn't consider the FEATURE_EH_FUNCLETS to be enabled for x86 and the funclets are actually the thing that uses the PSPSym. 
But since this is just a one line change, I won't block you on this. Once we start handling exceptions in complex nested try / catch scenarios, it will quickly show up whether it is better to enable the PSPSym or to fix some place in the stack walker instead.